doc/build.txt

```diff
@@ -18,6 +19,9 @@ all/the specified platforms.
     --emulator ......................... Deploy to an emulator
     --target ........................... Deploy to a specific target
 
+    --buildConfig....................... Use the specified build configuration
```

Comment:
It'd be great if you could document an example of how to use this flag in both build.txt and run.txt